node.go

```diff
@@ -218,6 +218,11 @@ func (n *Node) Description() string {
 	return n.meta.Description
 }
 
+// Returns the full description of the node.
```

Comment:
The documentation seems to be wrong. Although we try to document as many exported functions as possible, we do not try to cover them all. I think you safely remove the comment.

Reply:
oops, fixed c5d900f